README.md

```diff
@@ -24,6 +24,51 @@ Help on usage:
 docker run --rm --pull always ghcr.io/iot-inspector/unblob:latest --help
 ```
 
+### Using Nix
+
+Unblob can be built and run using [Nix](https://nixos.org). The Nix
+derivation installs all 3rd party dependencies:
+
+1. [Install and configure Nix](https://nixos.org/download.html)
+
+    a. Optionally enable the experimental features so that you don't
+       need to pass `--extra-experimental-features "nix-command
+       flakes"` them to `nix` command invocations:
+
+        ```console
+        $ cat > ~/.config/nix/nix.conf <<EOF
+        experimental-features = nix-command flakes
+        EOF
+        ```
+
+    b. Optional, use pre-built binaries from GitHub using [cachix](https://app.cachix.org/cache/unblob):
```

Comment:
We use `Optionally ....` above and `Optional, ....` here. We should make them looks like the same.

Maybe add a `[OPTIONAL]` prefix?